packages/main/src/themes/List.css

```diff
@@ -1,6 +1,7 @@
 :host(:not([hidden])) {
 	display: block;
 	max-width: 100%;
+	width: 100%;
```

Comment:
I will document it, you are right It might visually breaking for applications, I prefer it 100% wide as well. The native "ul" element is also 100% wide.